### PR TITLE
Increase MaxFrameLen to 16 MB

### PR DESCRIPTION
**Pull request by jcortejoso**
EigenDA current limit for Holesky (their documentation is currently outdated but the limit seems to be set to 16 MB based on the updated tests from [this PR](https://github.com/Layr-Labs/eigenda-proxy/pull/100)).

### CODE REVIEW

**Comment by jcortejoso** [comment]
Tested on Holesky with EigenDA as AltDA.

File: op-node/rollup/derive/frame.go [hunk 1]
```diff
@@ -8,10 +8,10 @@ import (
 	"io"
 )
 
-// Frames cannot be larger than 1 MB.
 // Data transactions that carry frames are generally not larger than 128 KB due to L1 network conditions,
 // but we leave space to grow larger anyway (gas limit allows for more data).
-const MaxFrameLen = 1_000_000
+// For AltDA, frames size can be larger. Setting to 16 MB as current blob limit for EigenDA.
+const MaxFrameLen = 16_000_000
```
Comment (palango) [inline op-node/rollup/derive/frame.go:14]: How is the final size decided? Does that happen in the node or in the batcher?

Comment (jcortejoso) [reply op-node/rollup/derive/frame.go:14]: This change applies to derivation pipeline, and affects only to the limit enforced to op-node. op-batcher has a cmdflag to set the batch size limit (`--max-l1-tx-size-bytes`) and this is not affected by `MaxFrameLen`.

16 MB was chosen as it is currently the limit of blob size on EigenDA Holesky.
This does not mean that the frame size used is 16MB, but op-node would accept frames up to that size.

**Review by palango — APPROVED** [review]
in line 88 there's a comment, which you might want to adapt as well.
```
	// Cap frame length to MaxFrameLen (currently 1MB)
```

**Pull request merged**